### PR TITLE
react update (15.x)

### PR DESCRIPTION
**Pull request by sertaconay**
I hope it's OK.

### CODE REVIEW

**Comment by magalhas** [comment]
Can you please squash both commits? :)

File: package.json [hunk 1]
```diff
@@ -17,15 +17,15 @@
   },
   "homepage": "https://github.com/magalhas/express-react-engine",
   "peerDependencies": {
-    "react": "^0.14.0",
-    "react-dom": "^0.14.0"
+    "react": "^15.0",
```
Comment (ljharb) [inline package.json]: perhaps this could be `"react": "14.x || 15.x"` and so on, so that it can remain compatible with both versions?

Comment (magalhas) [inline package.json]: It makes sense.

**Comment by pstadler** [comment]
I'm interested in getting this merged. You can now squash PRs directly see above ^

**Comment by magalhas** [comment]
I'll be releasing a new version with this pretty soon!

**Pull request merged**